parlai/agents/hugging_face/dialogpt.py

```diff
@@ -26,6 +26,12 @@ class DialoGPTDecoder(GPT2Decoder):
     This decoder is initialized with the pretrained model from Hugging Face.
     """
 
+    def __init__(self, opt, dict):
+        super().__init__(opt, dict)
+        if opt.get('batchsize', 1) == 1 and self.END_IDX == self.NULL_IDX:
```

Comment:
when is the latter condition not going to be true? if you are inheriting from this model but changing things?

Reply:
when ```-bs 1 --add_special_token True ``` ? basically I only want to override the NULL_IDX if it's the same as END_IDX.